packages/cozy-harvest-lib/src/components/KonnectorAccounts.jsx

```diff
@@ -92,7 +88,8 @@ export class KonnectorAccounts extends React.Component {
   }
 
   async fetchAccounts() {
-    const triggers = get(this.props, 'konnector.triggers.data', [])
+    const triggers =
+      this.props.triggers || get(this.props, 'konnector.triggers.data', [])
```

Comment:
this change should not be in the refactor commit. It can changes a lot the behavior 🤔

Reply:
Why, the triggers prop is new here, I do not see the risk here 🤔

Reply:
Didn't check exactly where {...props} is used, but : 
```
cozy-harvest-lib git grep -Ri "{...props}" . | wc -l
      32
```

Here is the risk I see. 

but after a quick check, `KonnectorAccounts` seems to be only used one time with only one prop, so it's fine (if we get this change outside of a refactor one)

Reply:
Ouch, I did not see the problem with prop drilling, thanks for reminding me not to use this technique :D 

Anyway, you're right for this not to be a refactor, I think I included it by mistake. I've splitted in two the commit, which was helpful also to put a nice description in the commit. See [`8f637cd` (#1067)](https://github.com/cozy/cozy-libs/pull/1067/commits/8f637cde9da4bd71f888cfdef49c607d007ee183)